wdl-lint/src/rules/blank_lines_between_elements.rs

```diff
@@ -452,6 +478,11 @@ fn flag_all_blank_lines_within(syntax: &SyntaxNode, state: &mut Diagnostics) {
             let count = c.to_string().chars().filter(|c| *c == '\n').count();
             if count > 1 {
                 state.add(excess_blank_line(c.text_range().to_span()));
+                // state.exceptable_add(
+                //     excess_blank_line(c.text_range().to_span()),
+                //     SyntaxElement::from(syntax.clone()),
+                //     BlankLinesBetweenElementsRule::exceptable_nodes(),
```

Comment:
But then once we leave the `impl` block by calling a utility function, I lose access to the `*Rule::exceptable_nodes()` function and now I'm not sure how to get that information to the `state`/`diagnostics` struct in `exceptable_add()`